pmd-ant/src/main/java/net/sourceforge/pmd/ant/Formatter.java

```diff
@@ -150,7 +150,6 @@ private static String unknownRendererMessage(String userSpecifiedType) {
         return sb.toString();
     }
 
-    // FIXME - hm, what about this consoleRenderer thing... need a test for this
```

Comment:
This comment was actually from 2006, where the method createRenderer actually had a boolean parameter called "consoleRenderer" :smile: